src/testing/assertionTemplate.ts

```diff
@@ -26,21 +26,19 @@ const findOne = (nodes: DNode | DNode[], selector: string): DNode | undefined =>
 		finalSelector = `[assertion-key='${selector.substr(1)}']`;
 		[node] = select(finalSelector, nodes);
 	}
-	return node;
-};
-
-type NodeWithProperties = (VNode | WNode) & { properties: { [index: string]: any } };
 
-const guard = (node: DNode): NodeWithProperties => {
 	if (!node) {
-		throw Error('Node not found');
+		throw Error(`Cannot find node with selector ${selector}`);
 	}
 	if (!isWNode(node) && !isVNode(node)) {
 		throw Error('Cannot set or get on unknown node');
```

Comment:
Hmm I don’t think we should change this. The findOne is meant to be equivalent to querySelector and not throw. It also doesn’t make sense to throw a setting unknown node here given the function name?

Reply:
Hehe, yeah, I wasn't sure about these changes. The other option is i can pass the selector in `guard` so it can use it in the error message, but thats going to end up looking silly and be a bit of a pain every time you call it.  Maybe i should just remove that last commit?

Reply:
Ha yeah I agree, maybe for now we just stick with the trigger change and we can raise a separate issue for making error messages more consistent (it might be that we use the same function in the harness and assertion template)